docsite-js/snackbar.js

```diff
@@ -0,0 +1,25 @@
+let toggleBaselineBtn = document.getElementById("toggle-baseline-snackbar");
+let toggleLeadingBtn = document.getElementById("toggle-leading-snackbar");
+let baselineSnackbar = document.querySelector(".baseline");
+let leadingSnackbar = document.querySelector(".leading");
+let closeBaselineSnackbar = document.getElementById("demo-baseline-snackbar-close");
+let closeLeadingSnackbar = document.getElementById("demo-leading-snackbar-close");
+
+toggleBaselineBtn.addEventListener("click",()=>{
+    baselineSnackbar.classList.toggle("show");
+    setTimeout(function(){ leadingSnackbar.classList.remove("show");}, 3000);
+})
+toggleLeadingBtn.addEventListener("click",()=>{
+    leadingSnackbar.classList.toggle("show");
+    setTimeout(function(){ baselineSnackbar.classList.remove("show");}, 3000);
```

Comment:
Plan to use ES6 syntax wherever possible.

Reply:
Good catch, refactored it.